tests/utils.go

```diff
@@ -2813,6 +2813,16 @@ func NewHelloWorldJobUDP(host string, port string) *k8sv1.Pod {
 	return job
 }
 
+// NewHelloWorldJobHttp gets an IP address and a port, which it uses to create a pod.
+// This pod tries to contact the host on the provided port, over HTTP.
+// On success - it expects to receive "Hello World!".
+func NewHelloWorldJobHttp(host string, port string) *k8sv1.Pod {
```

Comment:
Just out of curiosity why 'New' in the function name?

Reply:
Same naming convention as the other 2 functions that already exist: NewHelloWorldJob and NewHelloWorldJobUDP.